cylc/flow/network/__init__.py

```diff
@@ -22,6 +22,7 @@
 
 import zmq
 import zmq.asyncio
+import zmq.auth
```

Comment:
Just wondering why this import was added? zmq is already imported, is it more efficient to pull both in? I'm not fully sure of the intricacies of imports.

Reply:
zmq.auth is a module so not ordinarally available by `import zmq`.

```python
>>> import zmq
>>> zmq.auth
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'zmq' has no attribute 'auth'
>>> import zmq.auth
>>> zmq.auth
<module 'zmq.auth' from '/.../site-packages/zmq/auth/__init__.py'>

```

I think this worked before because zmq.auth was imported in another file so the namespace was already loaded into the `zmq` namespace when we imported `zmq` in this module? Not sure. Python's import system is pretty nasty.

Reply:
Ahhh ok, thanks, that's a bit strange - I imagine I've been importing incorrectly all this time 😆

Reply:
NP, easy to do, I didn't spot it before, neither did flake8 which is disappointing.